trustmanager/remoteks/client_test.go

```diff
@@ -0,0 +1,79 @@
+package remoteks
+
+import (
+	"crypto/tls"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+
+	"github.com/docker/notary/storage"
+)
+
+const addr = "127.0.0.1:9876"
+
+var insecureTLS = tls.Config{
+	InsecureSkipVerify: true,
+}
+
+func setupTestServer(t *testing.T) *grpc.Server {
+	s := grpc.NewServer()
+	st := NewGRPCStorage(storage.NewMemoryStore(nil))
+	l, err := net.Listen(
+		"tcp",
+		addr,
+	)
+	require.NoError(t, err)
+	RegisterStoreServer(s, st)
+	go func() {
+		err := s.Serve(l)
+		t.Logf("server errored %s", err)
+	}()
+	return s
+}
+
+func TestRemoteStore(t *testing.T) {
+	name := "testfile"
+	bytes := []byte{'1'}
+
+	s := setupTestServer(t)
+	defer s.Stop()
+
+	// can't just use NewRemoteStore because it correctly sets up tls
+	// config and for testing purposes it's easier for the client to just
+	// be insecure
+	cc, err := grpc.Dial(
```

Comment:
You are correct but it seems like `InsecureSkipVerify` and `WithInsecure` achieve the same thing so I figured it was easier to just use the well understood tls config and let users set that up however they please, rather than having 2 ways to set clients to insecure mode.